components/dashboard/assets.tsx

```diff
@@ -0,0 +1,26 @@
+import { defi } from "../../utils/tzktHooks";
+
+type AssetsProperty = {
```

Comment:
Généralement la convention de nommage pour les props c'est <nom>Props
Par exemple `AssetsProps`

Reply:
fixed in: https://github.com/marigold-dev/tzsafe-ui/pull/167/commits/5e90cea124fe094c16f6d29919b28da0fda59b27